common/cuda_hip/distributed/assembly_kernels.cpp

```diff
@@ -90,7 +90,7 @@ void count_non_owning_entries(
         num_parts, local_part, row_part_ptrs.get_data(), send_count.get_data());
 }
 
-GKO_INSTANTIATE_FOR_EACH_VALUE_AND_LOCAL_GLOBAL_INDEX_TYPE_BASE(
+GKO_INSTANTIATE_FOR_EACH_VALUE_AND_LOCAL_GLOBAL_INDEX_TYPE(
```

Comment:
The base version of that macro is now unused.

Reply:
I use base and the normal one is built on the base one.